<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempInvalidDuplicateEditDialog</class>
 <widget class="QDialog" name="TempInvalidDuplicateEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Дубликат документа временной нетрудоспособности</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="8" column="0">
    <widget class="QLabel" name="lblNote">
     <property name="text">
      <string>П&amp;римечание</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSerial">
     <property name="text">
      <string>Серия</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" rowspan="2" colspan="2">
    <widget class="QPlainTextEdit" name="edtNote"/>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edtNumber"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="CBlankComboBox" name="edtSerial">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblNumber">
     <property name="text">
      <string>&amp;Номер</string>
     </property>
     <property name="buddy">
      <cstring>edtNumber</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>&amp;Дата</string>
     </property>
     <property name="buddy">
      <cstring>edtDate</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>&amp;Выдал</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblDestination">
     <property name="text">
      <string>&amp;Для представления в</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="edtDestination"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbReason"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblReason">
     <property name="text">
      <string>&amp;Причина выдачи</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QCheckBox" name="chkInsuranceOfficeMark">
     <property name="text">
      <string>Отметка страхового стола</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="edtPlaceWork">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblPlaceWork">
     <property name="text">
      <string>Место работы</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="CDateEdit" name="edtDate"/>
   </item>
   <item row="2" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Председатель КЭР</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbExpert"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CBlankComboBox</class>
   <extends>QComboBox</extends>
   <header>Blank/BlankComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtNumber</tabstop>
  <tabstop>edtDate</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>edtDestination</tabstop>
  <tabstop>cmbReason</tabstop>
  <tabstop>edtPlaceWork</tabstop>
  <tabstop>edtNote</tabstop>
  <tabstop>chkInsuranceOfficeMark</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TempInvalidDuplicateEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TempInvalidDuplicateEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
